OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -115,6 +115,10 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
 // OUTPUTS
 //=============================================================================
     OpenSim_DECLARE_OUTPUT(power, double, calcPower, SimTK::Stage::Acceleration);
+    OpenSim_DECLARE_OUTPUT(reaction_child_origin_in_ground, SimTK::SpatialVec, 
+        calcReactionOnChildAtOriginInGround, SimTK::Stage::Acceleration);
+    OpenSim_DECLARE_OUTPUT(reaction_parent_origin_in_ground, SimTK::SpatialVec, 
```

Comment:
I'm perfectly fine with any name we agree upon, you probably have used these the most.